src/components/file/migrations/ChangeFileToFileRels.migration.ts

```diff
@@ -0,0 +1,44 @@
+import { node, relation } from 'cypher-query-builder';
+import { BaseMigration, Migration } from '../../../core';
+
+@Migration('2022-04-25T14:46:26')
+export class ChangeFileToFileRels extends BaseMigration {
+  async up() {
+    // first, invert the relationship from incoming to outgoing
+    const res = await this.db
+      .query()
+      .match([
+        node('parent', 'FileNode'),
+        relation('in', 'rel', 'parent'),
+        node('child'),
+      ])
+      .raw(`CALL apoc.refactor.invert(rel) yield input, output`)
```

Comment:
We also have around 500k of these relationships in prod. Any idea if either of these procedures will handle this? Or do we need to wrap them in an `apoc.periodic.iterate`?